packages/redux/src/actions.ts

```diff
@@ -3,6 +3,7 @@ import type { ThunkAction } from 'redux-thunk';
 import type { CommandHistoryStorage } from '@deephaven/console';
 import type { FileStorage } from '@deephaven/file-explorer';
 import type { dh as DhType } from '@deephaven/jsapi-types';
+import type { PluginModuleMap } from '@deephaven/plugin';
```

Comment:
Should `packages/redux` include @deephaven/plugin in package dependencies?

Reply:
I added it. I'm not entirely sure how the types would be affected by a consumer if they only installed redux, but since it's in the `tsconfig.json` for the package, I added it to dependencies